spec/features/borrower_creates_loan_request.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.feature "borrower creates loan request" do
+  let(:borrower) { User.create(name: "Richard",
+                               email: "richard@example.com",
+                               password: "password",
+                               role: "borrower")
+                   }
+
+  def create_loan_request(title, description, amount)
+    click_link_or_button 'Create Loan Request'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.